### ISSUE TITLE
Retina icons for preferences?

### ISSUE THREAD

**Issue by danielcompton** [erikdoe/ccmenu#88]
The current icons are non-retina:

<img width="352" alt="screenshot of 16-07-17 2-46-31 pm" src="https://user-images.githubusercontent.com/811954/28244117-9246b994-6a35-11e7-91eb-84e92b7eb2fb.png">

**Comment by erikdtw** [comment]
Agreed that it would be good to have hi-res icons. It's just that I'm not a graphic artist by any stretch of imagination. If someone contributes better icons I'll be more than happy to add them.

**Comment by erikdoe** [comment]
Agreed that it would be good to have hi-res icons. It's just that I'm not a graphic artist by any stretch of imagination. If someone contributes better icons I'll be more than happy to add them.

**Comment by erikdoe** [comment]
The way to go here, if I look at Big Sur, will be to use SF Symbol icons. I know how to get them out of the app so it's just a matter of time.
